dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -70,23 +73,23 @@ ImmutableList<Task<?>> createTasks(ConnectorArguments arguments, Duration querie
     builder.add(new DumpMetadataTask(arguments, scope.formatName()));
     builder.add(new FormatTask(scope.formatName()));
     for (String name : awrNames()) {
-      OracleStatsQuery item = OracleStatsQuery.createAwr(name, queriedDuration);
+      QueryGroup awr = QueryGroup.create(/* required= */ false, AWR, MULTI_TENANT);
+      OracleStatsQuery item = OracleStatsQuery.create(name, awr, queriedDuration);
       builder.add(StatsJdbcTask.fromQuery(item));
     }
     for (String name : nativeNames(/* required= */ true)) {
-      OracleStatsQuery item =
-          OracleStatsQuery.createNative(
-              name, /* isRequired= */ true, queriedDuration, MULTI_TENANT);
+      QueryGroup requiredGroup = QueryGroup.create(true, NATIVE, MULTI_TENANT);
```

Comment:
I would suggest adding the `/* required= */` parameter hint for the `true` literal.

Reply:
Done, I missed that one but it's fixed now